lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -67,7 +67,6 @@
                     // name is being routed to.
                     router = this.router = new Backbone.Router();
                     _.each([
-                        [':default', _.bind(this.routeNotFound, this)],
```

Comment:
@andy-armstrong Got it. The initial router was looking at `content` which was the previous skip-link target. I recently updated it to `main` so I updated the router to listen for `main` instead. Seems to be working. Updating the sandy so others can verify.